packages/react-components/react-theme/src/alias/darkColorPalette.ts

```diff
@@ -16,7 +16,7 @@ const statusColorPaletteTokens = statusSharedColorNames.reduce((acc, sharedColor
     [`colorPalette${color}Border2`]: statusSharedColors[sharedColor].tint20,
   };
 
-  return { ...acc, ...sharedColorTokens };
+  return Object.assign(acc, sharedColorTokens);
```

Comment:
(nit) These could be simplified even further to skip creating the `sharedColorTokens` object entirely, and just assign directly to `acc`. I don't think this would be a huge additional perf improvement, so it's optional to make this extra change.
```ts
acc[`colorPalette${color}Background1`] = statusSharedColors[sharedColor].shade40;
acc[`colorPalette${color}Background2`] = statusSharedColors[sharedColor].shade30;
acc[`colorPalette${color}Background3`] = statusSharedColors[sharedColor].primary;
acc[`colorPalette${color}Foreground1`] = statusSharedColors[sharedColor].tint30;
acc[`colorPalette${color}Foreground2`] = statusSharedColors[sharedColor].tint40;
acc[`colorPalette${color}Foreground3`] = statusSharedColors[sharedColor].tint20;
acc[`colorPalette${color}BorderActive`] = statusSharedColors[sharedColor].tint30;
acc[`colorPalette${color}Border1`] = statusSharedColors[sharedColor].primary;
acc[`colorPalette${color}Border2`] = statusSharedColors[sharedColor].tint20;
```